### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by felixpalmer**
Copied across the boilerplate files from the deck.gl repo and got a minimal build working. `yarn tsbuild` will output the library into `dist/`

The code in the library currently just includes the functions that are used in deck.gl and there are no tests/docs

### CODE REVIEW

**Review by alasarr — COMMENTED** [review]
I think would be better to have mono repo called quadbin with the implementation for multiple languages. I talked about this with @Jesus89 and he agrees. Thoughts @felixpalmer ?

**Comment by Jesus89** [comment]
We started with the mono repo idea, but @felixpalmer proposed having one repo per language. I think it has some advantages this way for pull requests, releases, and it's similar to what it's done in h3: h3-js, h3-py, h3-java, ...

**Comment by felixpalmer** [comment]
Copying my comment from the story: 

I don't feel that strongly about it, I was just taking inspiration from what H3 does. Having multiple repos has the advantage that Issues/PRs are specific to the language. Also if in the future someone wants to contribute an implementation in another language they will likely do it in their own repo and it feels cleaner to have one repo per language in that case.

**Comment by alasarr** [comment]
Not a strong opinion here, I understand your pros.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  "description": "Utility functions for working with Quadbins",
+  "license": "MIT",
+  "private": true,
+  "keywords": ["quadbin", "webgl", "visualization"],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/CartoDB/quadbin-js.git"
+  },
+  "scripts": {
+    "clean": "ocular-clean",
```
Comment (alasarr) [inline package.json:12]: I'd include ocular dependency here. As it's not related with vis.gl I'd use something more standard

Comment (felixpalmer) [reply package.json:12]: You mean you *wouldn't* include it?

**Pull request merged**